### ISSUE TITLE
Re-thinking R08 granularity

### ISSUE THREAD

**Issue by vturpin** [nvs-vocabs/ArgoVocabs#35]
The current R08 combines float platform/sensor. It is not a good way to go, as it is multiplying the need for new codes (e.g. instead of one code for APEX floats, we now have APEX with SBE, APEX with RBR, etc.). 

How can we rethink this approach, in collaboration with WMO people.

Contact and links : 
Anna Milan [amilan@wmo.int](mailto:amilan@wmo.int). https://github.com/wmo-im/CCT/issues/24
Current vocab : http://vocab.nerc.ac.uk/collection/R08/current/
Github R08 : https://github.com/nvs-vocabs/R08

**Comment by apswong** [comment]
We should remember that R08 is a subset of WMO code table 1770, which contains codes that describe the instruments make and type for transmitting their data on the GTS.  It does look like to fully describe an instrument, one would need to specify its platform (float/glider/XBT/etc) and its sensor (SBE/RBR/FSI/etc). However, it also looks like there are only about 60 unused codes left in WMO code table 1770.  So it'll be good to find a solution before we run out of codes.

I was told that people in charge of the BUFR format may have come up with a solution to separate the platform and the sensor.  It may help to include both WMO and BUFR people in this discussion.

**Comment by tcarval** [comment]
The R08 table is indeed useful for the GTS-BUFR distribution of Argo data.
Its revision depends on the WMO governance.
Ideally, WMO would make it obsolete and replace it with a platform table and a sensor table.
That is what Argo does with:
- platform type: table 23
- sensor type: table 25

If you agree Victor, this could be the Argo message to WMO : WMO code table 1770 (Argo R08)nshould be obsoleted and replaced with platform and sensors table (Argo R23 and R25)

**Comment by apswong** [comment]
I thought Argo R08 is more akin to R23+R27, not R23+R25.  We should double check this.

**Comment by vturpin** [comment]
Yes I totally agree Thierry.
We meet our WMO colleague next week. I will add this question on the agenda. "How can we influence WMO reference list?" with this case as an exemple.
I'll update on this thread !

**Comment by tcarval** [comment]
> 

You're right Annie, the sensor type table is R27 (R25 is more generic), I corrected my answer above

**Comment by vpaba** [comment]
Hi all @tcarval, @vturpin and @apswong, I know this is an old thread, though I was just wondering whether there are any updates on the future of WMO code table 1770. What is the current plan for this table from the WMO/BUFR side? Are WMO and BUFR still considering separating platforms and sensors, or are they exploring other options? Do they plan to use new or existing controlled vocabularies to manage this information? Any information would be greatly appreciated, Thank you
